tests/app/shared/issue-tables/issue-sorter.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { getSortedData } from '../../../../src/app/shared/issue-tables/issue-sorter';
+import { MatSort } from '@angular/material';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+
+describe('issuer-sorter', () => {
+  describe('getSortedData()', () => {
+    const dummyIssue: Issue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
+    const otherDummyIssue: Issue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_ASSIGNEES);
+    const issuesList: Issue[] = [dummyIssue, otherDummyIssue];
+    const matSort: MatSort = new MatSort();
+
+    it('sorts issues based on their string fields correctly', () => {
```

Comment:
Do you plan to add tests for the other sort keys? I think `assignees` and `Todo Remaining` should be tested, as they are slightly more complex.

Reply:
Yes, I will plan to do so!